### PR TITLE
Added translations for meetings and moments.

### PR DESCRIPTION
**Pull request by matcha440**
# Description 
Next round of Japanese translations - Meetings and Moments.

## Corresponding Issue

#1215 

# Test Coverage

✅

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -243,49 +243,49 @@ ja:
...
+      what_happened: '何が起こりましたか？'
+      viewers_hint: 'モーメントが見える仲間'
+      draft_question: 'このモーメントの下書きを保存したいですか？'
     index:
-      subtitle: 'Delve deep into your moments - events and situations that affect your mental health.'
-      instructions: 'You haven''t written about any moments yet.<br><br>Don''t know where to start? What''s something that happened today that you would like to share and discuss with a loved one? It can be positive or negative! You can don''t have to share it right away - only when you''re ready to! Here''s an example of a Moment.'
-      example_name: 'Panicking over interview tomorrow!'
-      example_category: Career
+      subtitle: 'あなたのメンタルヘルスに影響を与える出来事や状況を深く探る。'
+      instructions: 'まだモーメントについて何も書いていません。<br><br>最初は難しいですか？今日、あなたが大切な人と分かち合って話し合いたいことは何ですか？ポジテイブなことでもネガティブなことでも大丈夫です。モーメントを作成してもすぐに共有する必要はありません。モーメントの例はこちらです。'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      instructions: 'まだモーメントについて何も書いていません。<br><br>大切な人と話し合いたいことはありますか？ポジテイブなことでもネガティブなことでも大丈夫です。モーメントを作成してもすぐに共有する必要はありません。モーメントの例はこちらです。'
```

File: config/locales/ja.yml [hunk 2]
```diff
@@ -243,49 +243,49 @@ ja:
...
-      body: 'Here''s a friendly reminder that your meeting tomorrow, "%{meeting_name}", will be taking place at %{time} at %{location}!'
+      subject: '%{meeting_name}という会合は明日%{time}にあります。'
+      body: '忘れないでください。%{meeting_name}という会合は%{location}で%{time}に行います。'
     leave:
-      error: 'You cannot leave the meeting, you are the only leader.'
-      success: 'You have left %{meeting}'
-    new: 'New Meeting'
+      error: 'リーダーはあなたしかいませんので、会合を立ち去ることができません。'
+      success: '%{meeting}を立ち去りました。'
+    new: '新しい会合'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
    new: '新しい会議'
```

File: config/locales/ja.yml [hunk 3]
```diff
@@ -243,49 +243,49 @@ ja:
...
+      viewers_hint: 'モーメントが見える仲間'
+      draft_question: 'このモーメントの下書きを保存したいですか？'
     index:
-      subtitle: 'Delve deep into your moments - events and situations that affect your mental health.'
-      instructions: 'You haven''t written about any moments yet.<br><br>Don''t know where to start? What''s something that happened today that you would like to share and discuss with a loved one? It can be positive or negative! You can don''t have to share it right away - only when you''re ready to! Here''s an example of a Moment.'
-      example_name: 'Panicking over interview tomorrow!'
-      example_category: Career
+      subtitle: 'あなたのメンタルヘルスに影響を与える出来事や状況を深く探る。'
+      instructions: 'まだモーメントについて何も書いていません。<br><br>最初は難しいですか？今日、あなたが大切な人と分かち合って話し合いたいことは何ですか？ポジテイブなことでもネガティブなことでも大丈夫です。モーメントを作成してもすぐに共有する必要はありません。モーメントの例はこちらです。'
+      example_name: '明日の面接はやばいパニックになりそう！'
```
Comment (nshki) [inline config/locales/ja.yml]:
```suggestion
      example_name: '明日のインタビューについてパニック状態！'
```

File: config/locales/ja.yml [hunk 4]
```diff
@@ -243,49 +243,49 @@ ja:
...
+      example_category: キャリア
     show:
-      strategies: 'What strategies would help?'
+      strategies: 'どんな戦略がいいですか？'
```
[the remark this answers is not in the document]
Comment (nshki) [reply config/locales/ja.yml]: I'd say 作戦 for this case. 戦略 has a more combat-related connotation.

**Comment by julianguyen** [comment]
Can't help at all with these translations hahaha, but just wanted to swing by and say this is looking awesome. Thank you for working on this everyone!

**Pull request merged**